react/components/ProductSummaryName/ProductSummaryName.js

```diff
@@ -57,6 +58,7 @@ ProductSummaryName.defaultProps = {
 ProductSummaryName.propTypes = {
   /** Name schema props */
   showFieldsProps: PropTypes.object,
+  tag: PropTypes.string,
```

Comment:
Just to be clearer, add this `tag` prop with the default value of `'h1'` to the `defaultProps` defined just above this `propTypes` definition :)